Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/BackHandler.kt

```diff
@@ -44,6 +53,12 @@ fun backPressHandler(onBackPressed: () -> Unit, enabled: Boolean = true) {
     // Using onCommit guarantees that failed transactions don't incorrectly toggle the
     // remembered callback.
     onCommit(enabled) {
```

Comment:
As `highPriority` could also potentially change, it should be included in `onCommit`

```suggestion
    onCommit(enabled, highPriority) {
```